addons/project/models/company.py

```diff
@@ -16,4 +16,13 @@ class ResCompany(models.Model):
 
     def _compute_analytic_plan_id(self):
         for company in self:
-            company.analytic_plan_id = self.env['account.analytic.plan'].with_company(company)._get_default()
+            default_plan = self.env['ir.config_parameter'].with_company(company).sudo().get_param("default_analytic_plan_id_%s" % company.id)
+            company.analytic_plan_id = int(default_plan) if default_plan else False
+            if not company.analytic_plan_id:
+                company.analytic_plan_id = self.env['account.analytic.plan'].with_company(company)._get_default()
+
+    def write(self, values):
+        if 'analytic_plan_id' in values:
+            self.env['ir.config_parameter'].sudo().set_param("default_analytic_plan_id_%s" % self.id,
```

Comment:
I've seen now that the overwrite of `res.company().write` in account module allow multiple records. So here we should probably also do `for company in self:` and use `company.id`.